docs/source/ops/connect/helm.rst

```diff
@@ -0,0 +1,281 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _connect-helm-chart:
+
+Connect Helm Chart
+==================
+
+As of 1.18.0, we provide an Early Access version of the Connect Helm Chart for
+Entreprise Edition customers. This page contains documentation for that Helm
+chart.
+
+Credentials
+-----------
+
+Like all Entreprise Edition components, the Helm Chart is hosted on
+Artifactory. To get both the Helm chart itself and the Docker images it relies
+on, you will need Artifactory credentials. In the rest of thie document, we
+assume that ``$ARTIFACTORY_USERNAME`` refers to your Artifactory user name,
+whereas ``$ARTIFACTORY_PASSWORD`` refers to your Artifactory API key.
+
+Installing the Helm chart repository
+------------------------------------
+
+To let your local Helm installation know about the Daml Connect Helm chart, you
+need to add the repository with::
+
+  helm repo add daml \
+    https://digitalasset.jfrog.io/artifactory/connect-helm-chart \
+    --username $ARTIFACTORY_USERNAME \
+    --password $ARTIFACTORY_PASSWORD
+
+This will install the repository as ``daml``; you can then list the available
+versions with::
+
+  helm search repo --devel -l daml
+
+The ``--devel`` flag lets Helm know that you want to list prerelease versions
+(in the Semver sense). To avoid any confusion as to the production-readiness of
+the Helm chart, while the feature is in Early Access, only prerelease versions
+of the Helm chart will be available.
+
+Later on, you can update your local listing to match the Artifactory state
+with::
+
+  helm repo update
+
+And you can deploy the latest prerelease version with::
+
+  helm install dm daml/daml-connect --devel --values values.yaml
+
+where ``values.yaml`` is a YAML file that includes at least the
+``imagePullSecret`` key. See the rest of this page for other options in
+``values.yaml``, and the Helm documentation for related Helm usage.
+
+Setting up the ``imagePullSecret``
+----------------------------------
+
+The Helm chart relies on the production-ready Docker images for individual
+components that are part of the Entreprise Edition. Specifically, it expects a
```

Comment:
Just do a search n replace for `Entreprise` to `Enterprise` :joy:

Reply:
Yes, that's what I did. 🙂